### PR TITLE
Dockerfile: update `nim-docker-base` hash

### PR DESCRIPTION
**Pull request by ee7**
Before this PR:
https://hub.docker.com/layers/nim-docker-base/exercism/nim-docker-base/0ad461c92d478db8678d6b7f0e4bfc7d414902fd/images/sha256-53558d4a62be5c83028e123ad6be259c416ef614b149df1273f8195346d80047

With this PR:
https://hub.docker.com/layers/nim-docker-base/exercism/nim-docker-base/2bd80b871e44a83c2f3c7d65f29c624e88a663da/images/sha256-2c300f96f6f339042d511bfa93ebf8316a9aed9919ed8c2af9fb23ce35788763

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ee7** [comment]
I'll update and merge this after we merge the (non-user-facing) PRs in `nim-docker-base`.

**Review by ynfle — APPROVED** [review]

**Review by ynfle — APPROVED** [review]

**Pull request merged**
